src/components/revoGrid/styles/revo-grid.common.scss

```
@@ -5,57 +5,57 @@
   revogr-header {
     .header-rgRow {
       &.group {
-        $shadow-bottom: 0 -1px 0 0 var(--revo-grid-header-border) inset;
+        $shadow-bottom: 0 -1px 0 0 var(--rg-theme-header-border) inset;
         box-shadow: $shadow-bottom;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)